src/reset-local-database.sh

```diff
@@ -0,0 +1,160 @@
+if [[ -z "${DJANGO_SUPERUSER_EMAIL}" ]]; then
+  printf 'No DJANGO_SUPERUSER_EMAIL environment variable or [-e <email>] argument found. Using the default "admin@example.com".\n'
+  export DJANGO_SUPERUSER_EMAIL=admin@example.com
+fi
+
+# Prompt user to confirm reset.
+read -p $'Are you sure you want to wipe your local fv_be database? [Y/N] ' yn
+case $yn in
+  [Yy]* )
+    # Drop the existing database
+    printf '\n'
+    printf 'Removing existing fv_be database.\n'
+    dropdb -f fv_be
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Database removal failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Create the empty database
+    printf '\n\n'
+    printf 'Recreating empty fv_be database.\n'
+    createdb --username=postgres fv_be
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Database creation failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Delete any existing backend migrations files
+    printf '\n\n'
+    printf 'Removing existing backend migrations.\n'
+    find $PWD/src/firstvoices/backend/migrations/ -not -name '__init__.py' -delete
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Backend migration removal failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Delete any existing users migrations files
+    printf '\n\n'
+    printf 'Removing existing users migrations.\n'
+    find $PWD/src/firstvoices/users/migrations/ -not -name '__init__.py' -delete
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Users migration removal failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Make new users migrations
+    printf '\n\n'
+    printf 'Generating users migrations.\n'
+    python $PWD/src/manage.py makemigrations users
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Users migration generation failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Make new backend migrations
+    printf '\n\n'
+    printf 'Generating backend migrations.\n'
+    python $PWD/src/manage.py makemigrations backend
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Backend migration generation failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Run the new migrations
+    printf '\n\n'
+    printf 'Running migrations.\n'
+    python $PWD/src/manage.py migrate
+    retval=$?
+    if [ $retval -ne 0 ]; then
+      printf "Migration execution failed: exit code $retval\n"
+      exit $retval
+    fi
+
+    # Create a superuser using the DJANGO_SUPERUSER_USERNAME, DJANGO_SUPERUSER_PASSWORD, and DJANGO_SUPERUSER_EMAIL environment variables.
+    printf '\n\n'
+    printf 'Creating a superuser account using the DJANGO_SUPERUSER_USERNAME, DJANGO_SUPERUSER_PASSWORD, and DJANGO_SUPERUSER_EMAIL environment variables.\n'
+    python $PWD/src/manage.py createsuperuser --noinput --id $DJANGO_SUPERUSER_USERNAME
```

Comment:
Does this work with the current dev branch, where the user requires both an id and a username?